examples/gno.land/p/demo/uassert/uassert.gno

```diff
@@ -380,10 +380,37 @@ func Empty(t TestingT, obj interface{}, msgs ...string) bool {
 		if val != "" {
```

Comment:
```go
func isNumberEmpty(n interface{}) (isNumber, isEmpty bool) {
	switch n := n.(type) {
	// NOTE: the cases are split individually, so that n becomes of the
	// asserted type; thus the comparison `n == 0` is concrete, and 0 is
	// not converted to interface{} and assumed to be an int.
	case int:
		return true, n == 0
	case int8:
		return true, n == 0
	case int16:
		return true, n == 0
	case int32:
		return true, n == 0
	case int64:
		return true, n == 0
	case uint:
		return true, n == 0
	case uint8:
		return true, n == 0
	case uint16:
		return true, n == 0
	case uint32:
		return true, n == 0
	case uint64:
		return true, n == 0
	case float32:
		return true, n == 0
	case float64:
		return true, n == 0
	}
	return false, false
}
```

then use it for both, before the type switch.

Reply:
updated, please double check. thank u.